_includes/footer.html

```diff
@@ -53,11 +53,7 @@
 										<i class="fa fa-vk" aria-hidden="true"></i>
 									</a>
 								</li>
-								<li>
-									<a href="https://in.pinterest.com/pin/390476230184177529/">
-										<i class=" ti-pinterest" aria-hidden="true"></i>
-									</a>
-								</li>
+								
```

Comment:
Why this Line?

Reply:
@saarthakchats Please remove this blank line